src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -86,6 +88,10 @@ public CommandResult undo(Model model) throws CommandException {
                 .build();
     }
 
+    /**
+     * Returns true if command can be run in current application state. Command can run in HOME application state.
+     * @param applicationState The given application state.
```

Comment:
```suggestion

     * @param applicationState The given application state.
```
I think there should be a whitespace above `@param`